core/iwasm/libraries/thread-mgr/thread_manager.c

```diff
@@ -362,6 +362,9 @@ wasm_cluster_spawn_exec_env(WASMExecEnv *exec_env)
     wasm_module_inst_t module_inst = get_module_inst(exec_env);
     wasm_module_t module;
     wasm_module_inst_t new_module_inst;
+#if WASM_ENABLE_LIBC_WASI != 0
+    WASIContext* wasi_ctx;
```

Comment:
Coding guidelines check failed, should define as:
`WASIContext *wasi_ctx;` but not `WASIContext* wasi_ctx;`